### DISCUSSION TITLE
[Help] I am trying to figure out how to use the sprite code

### DISCUSSION THREAD

**Discussion by BestKitsune** [Monika-After-Story/MonikaModDev#8245] [QA - code and content creation (technical)] [answered]
I have been trying to figure out how to use the sprite code for dialogue so if someone could give me a detailed how-to-guide. I cannot figure out what I did wrong
Also I want to learn python but I can't find any good free course to try, so if you have any suggestions that would be great!
(MAS-Monika After Story)

**Comment by ThePotatoGuy** [comment, marked as answer]
[This page](https://github.com/Monika-After-Story/MonikaModDev/wiki/Dialogue-Coding#dialogue) has all the info for writing dialogue.

As for python, refer to my answer on #8227

**Reply by BestKitsune** [reply]
Thank you I will check this out!
